cmd/config.go

```diff
@@ -22,7 +22,8 @@ import (
 
 const (
 	configFile              = "Furyfile"
-	repoPrefix              = "git@github.com:sighupio/fury-kubernetes"
+	httpsRepoPrefix         = "git::https://github.com/sighupio/fury-kubernetes"
```

Comment:
furyctl prints the url when downloading a package, and using https you see this strange url that starts with `git::`, is a little bit misleading. Like this:

```shell
2020/01/03 09:39:53 downloading: git::https://github.com/sighupio/fury-kubernetes-on-premise.git//roles/sighup-base?ref=master -> vendor/roles/on-premise/sighup-base
```

What do you think of droping the `git::` when we print the url?

Reply:
Also double `//` are showed. 
We can delete both if we want ;)